tests/sqllab_tests.py

```diff
@@ -600,6 +605,7 @@ def test_api_database(self):
             {r.get("database_name") for r in self.get_json_resp(url)["result"]},
         )
         self.delete_fake_db()
+        example_db.expose_in_sqllab = False
```

Comment:
Perhaps this should be reset in a `finally` block in case this test raises?

Reply:
i moved it to a fixture